src/zenml/entrypoints/base_entrypoint_configuration.py

```diff
@@ -157,6 +157,7 @@ def _parse_arguments(cls, arguments: List[str]) -> Dict[str, Any]:
         Raises:
             ValueError: If the arguments are not valid.
         """
+
```

Comment:
I think something will complain about this...

Reply:
Waiting on the checks myself.